api/src/test/resources/test-hibernate.cfg.xml

```diff
@@ -8,5 +8,6 @@
         <!-- Reporting -->
         <mapping resource="ReportDesign.hbm.xml"/>
         <mapping resource="ReportRequest.hbm.xml"/>
+        <mapping resource="Condition.hbm.xml"/>
```

Comment:
Why's this needed? Where is this ressource to be found?

Reply:
To manage `condition`s data, in this case Chronic illnesses.